tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -381,8 +381,7 @@ describe('Test Fleet access with RBAC with "CUSTOM ROLES" and "GITREPOS" using "
 
       // CAN go to Continuous Delivery Dashboard and "list" gitrepos
       cy.accesMenuSelection('Continuous Delivery', 'Dashboard');
-      cy.get("div[data-testid='collapsible-card-fleet-local']").contains(repoName).should('be.visible');
-      cy.get("div[data-testid='collapsible-card-fleet-default']").contains(repoNameDefault).should('be.visible');
+      cy.get('div.fleet-dashboard-data').should('contain',repoName).and('contain',repoNameDefault);
```

Comment:
Just curious, extra space required between `'contain' and repoName` ?

Reply:
Same applies below.

Reply:
ok. will do. thanks